lib/internal/streams/whatwg.js

```diff
@@ -0,0 +1,4283 @@
+'use strict';
+
+/* eslint-disable no-use-before-define */
```

Comment:
What is used before it's defined?

Reply:
A couple of the classes. I get into refactoring for performance optimization I'll be seeing what can be moved around.